.github/workflows/tag.yml

```
@@ -123,22 +123,41 @@ jobs:
         run: |
           echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\/v/}
           echo ::set-output name=TAG::${GITHUB_REF/refs\/tags\//}
+
+      - name: Checkout Homebrew-tap
+        uses: actions/checkout@master
+        with:
+          repository: GitGuardian/homebrew-tap
+          token: ${{ secrets.PAT_GITHUB }}
+          path: ./tap
+
       - name: Checkout Homebrew-ggshield
         uses: actions/checkout@master
         with:
           repository: GitGuardian/homebrew-ggshield
           token: ${{ secrets.PAT_GITHUB }}
-          path: ./brew
+          path: ./ggshield
 
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           pip install ggshield homebrew-pypi-poet
-          poet -f ggshield | sed 's/Shiny new formula/Detect secret in source code, scan your repos and docker images for leaks/g' | sed '7a\  license "MIT"' > brew/Formula/ggshield.rb
+          poet -f ggshield | sed 's/Shiny new formula/Detect secrets in source code, scan your repos and docker images for leaks/g' | sed '7a\  license "MIT"' | tee brew/Formula/ggshield.rb tap/Formula/ggshield.rb
```

nit: can you wrap that line? It was already super long before, now it's worse :)

I haven't tested it, but something like this would be more readable:

```suggestion
          poet -f ggshield \
              | sed 's/Shiny new formula/Detect secrets in source code, scan your repos and docker images for leaks/g' \
              | sed '7a\  license "MIT"' \
              | tee brew/Formula/ggshield.rb tap/Formula/ggshield.rb
```